docs/config.md

```diff
@@ -114,47 +114,46 @@ ui.graph.style = "square"
 
 ### Display of commit and change ids
 
-```toml
-ui.unique-prefixes = "brackets"  # Does not rely on color
-```
-
-Whether to highlight a unique prefix for commit & change ids. Possible
-values are `styled`, `brackets` and `none` (default: `styled`).
+Can be customized by the `format_short_id()` template alias.
 
 ```toml
-ui.log-id-preferred-length = 6
+[template-aliases]
+# Highlight unique prefix and show at least 12 characters (default)
+'format_short_id(id)' = 'id.shortest(12)'
+# Just unique prefix
+'format_short_id(id)' = 'id.shortest()'
+# Show unique prefix and the rest surrounded by brackets
+'format_short_id(id)' = 'id.shortest(12).prefix() "[" id.shortest(12).rest() "]"'
+# Always show 12 characters
+'format_short_id(id)' = 'id.short(12)'
 ```
 
-Determines the number of characters displayed for `jj log` for change or commit
-ids. The default is 12. If the `ui.unique-prefixes` option is not set to `none`,
-this option will be ignored if the number of characters it specifies is
-insufficient to print the entire unique prefix of an id.
-
-This option can be convenient to set on a per-repository level.
-
 ### Relative timestamps
 
+Can be customized by the `format_timestamp()` template alias.
+
 ```toml
-ui.relative-timestamps = true
+[template-aliases]
+# Full timestamp in ISO 8601 format (default)
+'format_timestamp(timestamp)' = 'timestamp'
+# Relative timestamp rendered as "x days/hours/seconds ago"
+'format_timestamp(timestamp)' = 'timestamp.ago()'
 ```
 
-False by default, but setting to true will change timestamps to be rendered
-as `x days/hours/seconds ago` instead of being rendered as a full timestamp.
-
 ### Author format
 
+Can be customized by the `format_short_signature()` template alias.
+
 ```toml
-ui.log-author-format = 'username'
+[template-aliases]
```

Comment:
I'm curious, is `none` still possible? What happens if I set `'format_short_signature(signature)'` to `''`? (Will there be extra spaces in the log output?)

Reply:
Yuya can give a more authoritative answer, but I think you would not get empty spaces in the output, thanks to the `separate()` template function here: https://github.com/martinvonz/jj/blob/29792dab9b6de1aacc5491c707b6ee1c07e66ade/src/config/templates.toml#L9-L21

That template function adds spaces (or whatever separator was specified) between non-empty arguments only.

Reply:
Yep, `''` (an empty template) should be omitted by `separate()`. In future version, we might require `'""'` (an empty string template) and turn `jj log -T ''` into an error.